Cargo.toml

```diff
@@ -138,7 +138,7 @@ itertools = "0.10"
 jsonb = { git = "https://github.com/databendlabs/jsonb.git", rev = "8c8d2fc294a39f3ff08909d60f718639cfba3875", default-features = false }
 lazy_static = "1.4"
 local-ip-address = "0.6"
-loki-api = { git = "https://github.com/shuiyisong/tracing-loki", branch = "chore/prost_version" }
+loki-proto = { git = "https://github.com/GreptimeTeam/loki-proto.git", branch = "main" }
```

Comment:
We recommend using `rev` in the main branch to avoid unconscious version change.